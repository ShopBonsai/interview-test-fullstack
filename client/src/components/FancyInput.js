import React from 'react';
import './fancyInput.css'

const FancyInput = ({ onChange, placeholder = 'Placeholder' }) => (
  <div className="input-container">
    <input className="input-field" onChange={onChange} type="text" placeholder={placeholder} />
    <span className="focus-border"></span>
  </div>
)

export default FancyInput;