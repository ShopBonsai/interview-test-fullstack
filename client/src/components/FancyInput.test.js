import React from 'react';
import renderer from 'react-test-renderer';
import FancyInput from './FancyInput';

describe('FancyInput', () => {
  it('should render correctly', () => {
    expect(
      renderer.create(
        <FancyInput />
      )
    ).toMatchSnapshot();
  });
});