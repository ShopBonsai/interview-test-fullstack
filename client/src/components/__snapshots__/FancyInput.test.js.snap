// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FancyInput should render correctly 1`] = `
<div
  className="input-container"
>
  <input
    className="input-field"
    placeholder="Placeholder"
    type="text"
  />
  <span
    className="focus-border"
  />
</div>
`;
